doc/make_tables.py

```diff
@@ -239,9 +239,7 @@ class ColorSchemeTable(DocTable):
     @classmethod
     def fetch_data(cls):
         # Fetch table data from ``COLOR_SCHEMES`` dictionary.
-        return [
-            {"scheme": cs, **data} for (cs, data) in pv.charts._MultiCompPlot.COLOR_SCHEMES.items()
-        ]
+        return [{"scheme": cs, **data} for (cs, data) in pv.colors.COLOR_SCHEMES.items()]
```

Comment:
@dcbr, any idea why `pv` is actually the `pyvista.plotting` namespace?

Reply:
This is happening in the `test_charts.py` module as well. Super weird....

Reply:
@adeak, perhaps something funny is going on with star imports and this might get resolved in #3764?

Reply:
I'm on mobile right now, but `pv` should by `pyvista`. But a bunch of nested modules get pulled into the main namespace, such as `charts` and `colors`. Part of the planned discussion on the linked PR is to decide what to get rid of. First step is to preserve the names without star imports (which is mostly done).

Reply:
No idea why this is happening, but this has been a thing since I first started "messing around" with pyvista, so it's definitely not a recent thing. For some reason all plotting modules are part of the global scope, so `pyvista.plotting` actually refers to the `pyvista.plotting.plotting` module.

Reply:
This is the first I've noticed this. We'll have to look into this as a part of #3764 or separately